ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml

```diff
@@ -0,0 +1,126 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  bgp_peer_groups:
+    documentation_options:
+      filename: Fabric Variables
+      table: BGP Peer Groups
+    description: |
+      Leverage an Arista EOS switch to generate the encrypted password using the correct peer group name.
+      Note that the name of the peer groups use '-' instead of '_' in EOS configuration.
+    type: dict
+    keys:
+      ipv4_underlay_peers:
+        type: dict
+        keys:
+          name:
+            type: str
+            default: IPv4-UNDERLAY-PEERS
+            description: Name of peer group.
+          password:
+            type: str
+            description: Encrypted password.
```

Comment:
for now we only support type 7

Reply:
addressed in [f94bf27](https://github.com/aristanetworks/ansible-avd/pull/2507/commits/f94bf272c4edf2aca1a925f2d2bf2469136eb45b)